x/bank/types/key.go

```diff
@@ -22,10 +22,17 @@ const (
 
 // KVStore keys
 var (
-	BalancesPrefix = []byte("balances")
-	SupplyKey      = []byte{0x00}
+	BalancesPrefix      = []byte("balances")
```

Comment:
Unrelated to this PR, but is there a reason we can't use a single byte prefix here too? @alexanderbez ?